.github/workflows/main.yml

```diff
@@ -34,17 +41,25 @@ jobs:
         go-version: ${{ matrix.go-version }}
 
     - name: Install protoc-gen-go
+      if: runner.os == 'Linux'
       run: |
         sudo env "GOBIN=/usr/bin" go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
 
     - name: Test go-criu
+      if: runner.os == 'Linux'
       run: sudo -E make test
 
     - name: Test magicgen script
+      if: runner.os == 'Linux'
       run: sudo -E make -C scripts/magic-gen test
 
     - name: Test CRIT
+      shell: bash
```

Comment:
I'm not sure we need this change - the vanilla shell should support comparisons out of the box

Reply:
Because Windows’ default shell uses a different scripting style, I added that line to support Windows. While it would be possible to split tests into Windows and non-Windows sections, I felt this approach was the cleanest, so I wrote it this way.

Reply:
That makes sense, thanks!